lute/parse/registry.py

```diff
@@ -4,57 +4,69 @@
 List of available parsers.
 """
 
+from importlib.metadata import entry_points
+
 from lute.parse.base import AbstractParser
 from lute.parse.space_delimited_parser import SpaceDelimitedParser, TurkishParser
 from lute.parse.mecab_parser import JapaneseParser
 from lute.parse.character_parser import ClassicalChineseParser
 
 
-# List of ALL parsers available, not necessarily all supported.
-# This design feels fishy, but it suffices for now.
-parsers = {
+__LUTE_PARSERS__ = {
```

Comment:
Any reason for the change ?  I don't disagree, I just don't follow the motivation.  More expressive/indicative of its intent?  (I'm not a big fan of how the parsers array is managed.)

Reply:
@jzohrab It makes the code easier to read because the module wide variable is obvious, and it also make classes importing the variable directly look odd, because it they should use a separate access method to prevent any unintended runtime changes to the parsers list.

Reply:
Makes sense, thanks.